pkgs/by-name/mo/mozillavpn/package.nix

```
@@ -23,27 +22,15 @@
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "mozillavpn";
-  version = "2.24.3";
+  version = "2.25.0";
   src = fetchFromGitHub {
     owner = "mozilla-mobile";
     repo = "mozilla-vpn-client";
     rev = "v${finalAttrs.version}";
```

```suggestion
    tag = "v${finalAttrs.version}";
```

Please consider using `tag` instead of `rev`.

---

Switched.